test/unittest/tests/utils_extlist.h

```diff
@@ -15,32 +15,32 @@
 
 namespace utils {
 /*!
- * An offset (_off) and a size (_size) of a WT_EXT for two use cases: specifying a test setup or an
+ * An offset (off) and a size (size) of a WT_EXT for two use cases: specifying a test setup or an
  * expected result.
  */
 struct off_size {
-    wt_off_t _off;
-    wt_off_t _size;
+    wt_off_t off;
+    wt_off_t size;
 
-    off_size(wt_off_t off = 0, wt_off_t size = 0) : _off(off), _size(size) {}
+    off_size(wt_off_t off = 0, wt_off_t size = 0) : off(off), size(size) {}
     /*!
      * end --
-     *     Return the end of the closed interval represented by _off and _size.
+     *     Return the end of the closed interval represented by off and size.
      */
     wt_off_t
     end(void) const
     {
-        return (_off + _size - 1);
+        return (off + size - 1);
     }
 };
 
 /*!
- * A test (_off_size) and the expected value (_expected_list) for operations that need an off_size
+ * A test (off_size) and the expected value (expected_list) for operations that need an off_size
  * to modify a WT_EXTLIST.
  */
 struct off_size_expected {
-    off_size _off_size;
-    std::vector<off_size> _expected_list;
+    off_size an_off_size;
```

Comment:
`an_off_size` is an odd name. How about something like `expected_off_size`? I think it would be more readable, and it would be more consistent with the `expected_list` field below.

Reply:
expected_off_size would be incorrect.  off_size is specifying what to test, expected_list is specifying the expected results. I tried to make clear in the comment preceding the struct which members specified the test, and which members specified the expected result.  Apparently I did not do that well.

When the struct fields had _ I could use _off_size for the test parameters.  That doesn't work when _ was removed.

I changed it to test_off_size.

Done